docs/advanced_topics/embeds.rst

```diff
@@ -221,6 +221,11 @@ the App ID and App Secret from your app:
             'class': 'wagtail.embeds.finders.instagram',
             'app_id': 'YOUR INSTAGRAM APP_ID HERE',
             'app_secret': 'YOUR INSTAGRAM APP_SECRET HERE',
+        },
+        
+        # Handles all other oEmbed providers the default way
```

Comment:
Thanks @drimacus182!

```suggestion
        # Handle all other oEmbed providers the default way
```

Let's use the standard imperative mood :)

Reply:
Just copied a line from the section above

Reply:
@tomdyson Are there any further steps I need to make to merge this into master?